syntax="proto2";

package sogou.nlu.rpc.example;

option java_generic_services = true;
option cc_generic_services = true;
option py_generic_services = true;

// /usr/local/protobuf3/bin/protoc --proto_path=. --python_out=./ ./echo.proto

message EchoRequest {
	required string message = 1;
}

message EchoResponse {
	required string message = 1;
}

service EchoService  
{
	rpc echo(EchoRequest) returns(EchoResponse);
}
